.github/workflows/copy_to_gitbook_repo.yml

```
@@ -35,7 +37,7 @@ jobs:
           user_email: '41898282+github-actions[bot]@users.noreply.github.com'
           user_name: 'github-actions[bot]'
           commit_message: |
-            Sync: ${{ github.event.head_commit.message }}
+            Sync ${{ inputs.action_name }}: ${{ github.event.head_commit.message }}
```

I wanted show what part of docs is being synced. snap/sso/common. Do you think it's redundant?

---

I think that should already be clear enough from the commit content itself, so I'd consider it redundant in the commit message, given the limited length

---

Or perhaps to not break the semantic commits from the original commits and not take up the precious non-truncated space with texts with little informational value, what about 
```suggestion
            ${{ github.event.head_commit.message }} (synced with ${{ inputs.action_name }})
```